### PR TITLE
Getting requirements set up better

### PR DESCRIPTION
**Pull request by dmclain**
switch to released versions and add a dev.txt

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "armstrong.apps.content",
+    "name": "armstrong.apps.articles",
```
Comment (tswicegood) [inline package.json:2]: Gah!  I can't believe I missed that.

**Comment by tswicegood** [comment]
Testing this now in a clean virtualenv.

**Comment by tswicegood** [comment]
This still needs a `fabric>=1.1` line in its `requirements/dev.txt`.

**Comment by dmclain** [comment]
and boom goes the dynamite

**Comment by tswicegood** [comment]
`armstrong.dev` should also be there.

As should [these lines](https://github.com/armstrong/armstrong.cli/blob/master/armstrong/cli/templates/standard/requirements/project.txt#L3-5) from `armstrong.cli`:

```
# This is included here because the latest release of django-mptt does not have
# the required code.
-e git://github.com/django-mptt/django-mptt.git@0af02a95877041b2fd6d458bd95413dc1666c321#egg=django_mptt-dev
```

Unfortunately, there's no way to declare a dependency on a Git repo properly. :-(

**Pull request merged**